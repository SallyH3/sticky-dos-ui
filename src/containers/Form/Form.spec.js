import React from 'react';
import { shallow } from 'enzyme';
import { Form } from './index.js';

describe('Form', () => {
  let wrapper;

  beforeEach(() => {
    wrapper=shallow(
      <Form />
    )
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });

  it.skip('should handle title change', () => {

  });

 	it.skip('should handle type change', () => {

  });

  it.skip('should handle string change', () => {

  });

  it.skip('should handle list change', () => {

  });

  it.skip('should add list input', () => {

  });

  it.skip('should remove list input', () => {

  });

  it.skip('should handle submit', () => {

  });

  it.skip('should store card', () => {

  });

  
});