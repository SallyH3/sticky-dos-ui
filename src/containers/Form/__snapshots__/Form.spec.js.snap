// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should match snapshot 1`] = `
<section
  className="Form__Container"
>
  <button
    className="Form__changeType"
    onClick={[Function]}
  >
    <i
      className="far fa-check-square"
      title="Click to change type of note"
    />
  </button>
  <form
    onSubmit={[Function]}
  >
    <input
      className="Form__input-title"
      maxLength="50"
      name="title"
      onChange={[Function]}
      placeholder="Enter Title"
      type="text"
      value=""
    />
    <fieldset
      className="Form__input-content"
    >
      <span
        key="1559079395176"
      >
        <input
          id="0"
          key="0"
          name="list"
          onChange={[Function]}
          placeholder="Enter List Item"
          type="text"
        />
        <i
          className="far fa-plus-square"
          id="0"
          key="i-0"
          onClick={[Function]}
        />
      </span>
    </fieldset>
    <input
      className="Form__submit"
      type="submit"
      value="SAVE"
    />
  </form>
</section>
`;
